packages/@webex/plugin-meetings/src/controls-options-manager/util.ts

```diff
@@ -184,6 +202,22 @@ class Utils {
     return Utils.hasHints({requiredHints, displayHints});
   }
 
+  public static canUpdateVideo(
+    control: ControlConfig<VideoProperties>,
+    displayHints: Array<string>
+  ) {
+    const requiredHints = [];
+
+    if (control.properties.enabled === true) {
+      requiredHints.push(DISPLAY_HINTS.ENABLE_VIDEO);
+    }
+    if (control.properties.enabled === false) {
+      requiredHints.push(DISPLAY_HINTS.DISABLE_VIDEO);
+    }
+
+    return Utils.hasHints({requiredHints, displayHints});
```

Comment:
This seems to be repeated with only the hints differing. Maybe a generic util function that also takes the two hints would be useful?

Reply:
I considered this, but decided against it [to handle booleans] once i realized there are some without this state [share_control].